[LC-325] Edit logic to manage the duration of leader complaint.

hyerim-kim opened the pull request:
- The duration of leader complaint belongs to `Epoch` and increases on every round until max duration.

hyerim-kim committed:
[LC-325] Edit logic to manage the duration of leader complaint.

- The duration of leader complaint belongs to `Epoch` and increases on every round until max duration.

zsaladin commented on loopchain/blockchain/epoch.py:
```diff
@@ -46,6 +46,7 @@ def __init__(self, block_manager, leader_id=None):
         self.__complain_vote = dict()  # complain vote dict { round : Vote }
         self.__complain_vote[self.round] = Vote(Epoch.COMPLAIN_VOTE_HASH, ObjectManager().channel_service.peer_manager)
         self.complained_result = None
+        self.complain_duration = conf.TIMEOUT_FOR_LEADER_COMPLAIN
```
How about calculating each time instead of storing value?

```python
@property
def complain_duration(self):
    return (2 ** self.round) * conf.TIMEOUT_FOR_LEADER_COMPLAIN
```

hyerim-kim committed:
[LC-325] Edit logic to figure the `complain_duration` out.

- Figure out `complain_duration` every using that instead of storing to a variable.
